.gitignore

```diff
@@ -1,4 +1,44 @@
-/.vscode/
+# OSX trash
+.DS_Store
+
+# Eclipse files
```

Comment:
We should either (1) remove this section or (2) include a complete list of eclipse-related files. https://github.com/github/gitignore/blob/master/Global/Eclipse.gitignore

I would prefer option 1 without any eclipse users chiming in.